src/TodoListMain.tsx

```diff
@@ -1,6 +1,7 @@
 import { useState } from "react";
 import Button from "./components/Button/Button";
 import Input from "./components/Input/Input";
+import TodoItemList from "./components/TodoItemList/TodoItemList";
 
 interface Todo {
```

Comment:
So this interface already exists in your new `TodoItemList.tsx` file. Let's figure out how to not repeat code by exporting this from that file and importing it in this file along with the `TodoItemList`.

Reply:
Made this harder than it had to be but I exported and imported it